apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -790,23 +790,8 @@ function getGutenboardingStatus( calypsoPort ) {
 		[ port2 ]
 	);
 	port1.onmessage = ( { data } ) => {
-		const {
-			isGutenboarding,
-			isSiteUnlaunched,
-			launchUrl,
-			isNewLaunchMobile,
-			isExperimental,
-			isPersistentLaunchButton,
-			isFocusedLaunchFlow,
```

Comment:
Let's not remove this yet since it's still in use: https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-button/index.ts#L71

Reply:
Resolving the conversation, since the use of the `isFocusedLaunchFlow` is being removed with this same PR now